src/js/components/Grommet/Grommet.js

```diff
@@ -50,6 +51,7 @@ const Grommet = forwardRef((props, ref) => {
     full,
     containerTarget = typeof document === 'object' ? document.body : undefined,
     theme: themeProp,
+    options = {},
```

Comment:
I think this will cause the context to be updated every time. If we create a `const defaultOptions = {};` outside of `Grommet`, I think that will avoid that.